C/services/notification/notification_manager.cpp

```diff
@@ -1230,14 +1230,17 @@ bool NotificationManager::APIcreateEmptyInstance(const string& name)
 			 "\"enumeration\", \"options\": [ \"one shot\", \"retriggered\", \"toggled\" ], "
 			 "\"displayName\" : \"Type\", \"order\" : \"4\","
 			 "\"default\" : \"one shot\"}, "
+		   "\"text\": {\"description\": \"Text message to send for this notification\", "
```

Comment:
Option to provide value for message is not available during creating Notification instance. It appears later on the GUI once Notification instance is created. It leads to confusion. Please rectify it.

Reply:
Can we please log this issue?

Reply:
@praveen-garg following JIRA has been logged https://scaledb.atlassian.net/browse/FOGL-10239